canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -237,7 +237,7 @@ $toolbar-divider-width: 1px;
 	}
 
 	& .toolbar-right-chevron {
-		padding: 9px 0 2px;
+		padding: 4px 0 0;
```

Comment:
@tomlyn This will affect the main toolbar as well
<img width="1725" alt="image" src="https://github.com/elyra-ai/canvas/assets/1628100/114d9b95-3590-4053-a368-09af8856c1ec">

Reply:
Good catch - that is because the context toolbar has smaller height for the menu items than the main toolbar.